src/rviz/visualizer_app.cpp

```diff
@@ -172,7 +172,7 @@ bool VisualizerApp::init( int argc, char** argv )
       if (vm.count("display-config"))
       {
         display_config = vm["display-config"].as<std::string>();
-        if( display_config.substr( display_config.size() - 4, 4 ) == ".vcg" )
+        if (display_config.size() >= 4 and display_config.substr( display_config.size() - 4, 4 ) == ".vcg" )
```

Comment:
Please use `&&`. Also, I know the rviz style is weird, but please preserve the whitespace when possible, should be `if ( dis...` if you like.

Reply:
Shall I switch to `if ( dis...` even if (almost) the entire rest of the file does not use this style or shall I reformat the file this way?

Reply:
Nah, that's ok, it's pretty inconsistent in this version anyways.